### PR TITLE
add phred-offset to rnaspades

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/rnaspades/rnaspades.xml [hunk 1]
```diff
@@ -93,6 +95,10 @@
         </repeat>
         <param optional="true" format="fasta,fastq" label="Trusted contigs" multiple="true" name="trusted_contigs" type="data" />
         <param optional="true" format="fasta,fastq" label="Untrusted contigs" multiple="true" name="untrusted_contigs" type="data" />
+         <param argument="--phred-offset" name="phred_offset" type="select" style="radio" label="Phred-Offset" help="Specify which type of FASTQ format you have. By default fastqsanger is assumed." >
```
Comment (mvdbeek) [inline tools/rnaspades/rnaspades.xml]: You don't want to just set this based on the data type ? That is the only reason we discriminate between fastqsanger/fastqillumina etc.

Comment (bgruening) [reply tools/rnaspades/rnaspades.xml]: The datatype is currently only "fastq". I was hopeing for some feedback from @davebx or any other expert with this tool. We would need to remove "fastq" from the datatypes in this case and include fastqsanger and fastqillumina explicitly ... which poeses all the other issues which we already know :)

Comment (bgruening) [reply tools/rnaspades/rnaspades.xml]: I'm happy to change it.

**Comment by bernt-matthias** [comment]
What will be the value of the data set's `ext` attribute if a fastqsanger or Illumina file is given?

**Pull request closed without merging**